lib/worker.js

```diff
@@ -292,40 +292,31 @@ class AssetComputeWorker {
         }
 
         // post-processing // TODO: Refactor this to share partas with post-processing in `processRendition` func
+        // eslint-disable-next-line prefer-const
         for (let rendition of this.renditions) {
-            // start post-processing timer
-            const postProcessingTimer = this.timers.currentPostProcessing = new Timer();
```

Comment:
I think we still need to keep the timer's reference `this.timers.currentPostProcessing` somewhere - wdyt @jdelbick ?